notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/bert_optimized_online_prediction.ipynb

```diff
@@ -1510,7 +1510,7 @@
       "source": [
```

Comment:
<p>use this form:</p><p> </p><pre class="ql-syntax" spellcheck="false"># Set this to true only if you'd like to delete your bucket
delete_bucket = False


if delete_bucket or os.getenv("IS_TESTING"):
&nbsp; &nbsp; ! gsutil rm -r $BUCKET_URI
<code></code>`
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/630/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='135'/>

Reply:
<p>done</p>